src/Sylius/Bundle/ShopBundle/Resources/translations/messages.en.yml

```diff
@@ -10,3 +10,4 @@ sylius:
                 personal_information: Personal information
                 change_password: Change password
                 order_history: Order history
+                address_book: Address book
```

Comment:
If you're changing this file, maybe consider sorting this messages alphabetically? :)